src/licensedcode/data/rules/independent-module-linking-exception_1.RULE

```diff
@@ -0,0 +1,23 @@
+---
+license_expression: independent-module-linking-exception
+is_license_text: yes
+---
+
+This is the file COPYING.FPC, it applies to the Free Pascal Run-Time Library 
+(RTL) and packages (packages) distributed by members of the Free Pascal 
+Development Team.
+
+The source code of the Free Pascal Runtime Libraries and packages are 
```

Comment:
As with https://github.com/aboutcode-org/scancode-toolkit/pull/4081/files#diff-4da17ca6a6ecd5c224ab03bb038aebd541a84565b1910645ed9bed7dae26e828 this should be an LGPL with exception, not just the exception.